README.md

```diff
@@ -166,6 +168,29 @@ You can import word templates by adding `<documenttemplate>` elements.
 </templateconfig>
 ```
 
+### Mailboxes
+
+#### Update Approve Test and Enable shared mailboxes
+
+You can update shared mailboxes with target email address, approve and test&enable by adding `<mailboxes>` elements.
+
+Environment variable name for the target environment must be "PACKAGEDEPLOYER_SETTINGS_ENVIRONMENT_PREFIX".
+
+**Environment variable**
+
+```powershell
```

Comment:
This is at odds with what we've got at the moment. I would recommend that anything that changes per environment (e.g. connections for connection references is a good example) is not stored in the ImportConfig.xml but rather passed at run-time.

Reply:
Agreed. I believe when discussed previously it was suggested to pass a CSV files . I'm not sure how you would use environment variables since it would be the format `PACKAGEDEPLOYER_SETTINGS_MAILBOX_{source email address}`= {target email}.

Reply:
that's pretty much what we've got for connection references 😄 not pretty but I believe the standard characters for emails (e.g. '@' and '.' etc.)  are valid for environment variable names. Passing them as `RuntimeSettings` looks a bit better

Reply:
@ewingjm @tdashworth - Thanks both. I will update code to support RuntimeSettings and remove the config from importconfig.